pkg/tagger/metrics.go

```diff
@@ -0,0 +1,22 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+package tagger
+
+import "github.com/DataDog/datadog-agent/pkg/telemetry"
+
+var (
```

Comment:
wondering if we could also add cache miss and partial cache miss metrics, as have this comment in tagger.Tag
```
	// Else, partial cache miss, query missing data
	// TODO: get logging on that to make sure we should optimize
```
I thought it could be interesting to cache miss rate, wdyt?

Reply:
After discussing with @ahmed-mez we found that getting accurate numbers on cache misses is not possible without larger changes. That's because some fetchers will always cache miss for entity types they don't know about (for instance, the docker fetcher will always cache miss for pods), polluting the metrics. We'll address that before introducing the metric.